docs/04-guides/get-viewer-engagement/02-api.mdx

```diff
@@ -29,25 +29,74 @@ feature requests related to metrics.
 ## Step 1: Register views
```

Comment:
IMO this step must be outside of the tabbed display. It is not an instruction for the "API", it is necessary whether you want to see the views on the dashboard or on the API, so it must be included even on the dashboard docs.

I guess it would be OK to just repeat this on the dashboard tab, but IMO ideally, it would come before either tab on the doc, then the tabbed display only shows how to view the metrics on dashboard vs api (but on both cases already have it registered).

I remember having commented about this already on another PR so sorry if I'm being repetitive 🤔

Reply:
agreed - I've updated accordingly!